### PR TITLE
[ZEPPELIN-939] Improve notebook authorization documentation

### PR DESCRIPTION
**Pull request by AhyoungRyu**
### What is this PR for?
Currently Zeppelin provides authorization mechanism on each notebooks. But it seems many users can not get much useful information through [the existing notebook authorization docs](https://zeppelin.apache.org/docs/0.6.0-incubating-SNAPSHOT/security/notebook_authorization.html). So I add some information so that users can follow step by step. 

Moreover,  [interpreter authorization docs](https://zeppelin.apache.org/docs/0.6.0-incubating-SNAPSHOT/security/interpreter_authorization.html) doesn't provide much information so far. This can be confused to users. So I removed it temporally. We can add it again when we have specific(?) feature for `interpreter & data source authorization`.

### What type of PR is it?
Improvement | Documentation

### Todos
* [x] - Remove security_overview.md & interpreter_authorization.md
* [x] - Improve notebook authorization docs

### What is the Jira issue?
[ZEPPELIN-939](https://issues.apache.org/jira/browse/ZEPPELIN-939)

### Screenshots (if appropriate)
 - **Before**
screen shot 2016-06-01 at 5 46 22 pm

 - **After**
screen shot 2016-06-01 at 5 48 17 pm
screen shot 2016-06-01 at 5 48 31 pm

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

File: docs/security/interpreter_authorization.md [hunk 1]
```diff
@@ -1,34 +0,0 @@
...
-
-## Data Source Authorization
-
-Data source authorization involves authenticating to the data source like a Mysql database and letting it determine user permissions.
-
-For the Hive interpreter, we need to maintain per-user connection pools.
-The interpret method takes the user string as parameter and executes the jdbc call using a connection in the user's connection pool.
-
-In case of Presto, we don't need password if the Presto DB server runs backend code using HDFS authorization for the user.
-For databases like Vertica and Mysql we have to store password information for users.
```
Comment (felixcheung) [inline docs/security/interpreter_authorization.md]: I agree we should fine tune this description, but is it more confusing that it is absent?

Comment (AhyoungRyu) [inline docs/security/interpreter_authorization.md]: @felixcheung Thanks for sharing your opinion. Yeah you're right. I also saw some informations is added in this docs at #860. So I'll revert this docs then.

**Comment by AhyoungRyu** [comment]
@felixcheung I added `interpreter_authorization.md` file again.

File: docs/security/notebook_authorization.md [hunk 2]
```diff
@@ -17,21 +17,39 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
...
+* Readers ( users or groups )
+* Writers ( users or groups )
 
-To initialize and modify note permissions, we provide UI like "Interpreter binding". The user inputs comma separated entities for owners, readers and writers.
-We execute a rest api call with this information. In the backend we get the user information for the connection and allow the operation if the user and groups
-associated with the current user have at least one entity that belongs to owner entities for the note.
+<center><img src="../assets/themes/zeppelin/img/docs-img/permission_setting.png"></center>
+
+Fill out the each forms with comma seperated **users** and **groups** configured in `conf/shiro.ini` file.
+If the form is empty (*), it means that any users can perform that operation.
```
Comment (felixcheung) [inline docs/security/notebook_authorization.md]: is that empty or "*"?

**Comment by felixcheung** [comment]
minor question, looks good

File: docs/security/notebook_authorization.md (hunk 2 above)
Comment (AhyoungRyu) [inline docs/security/notebook_authorization.md]: @felixcheung The default value is set as `*`. It means a user leave the text box empty.
Here is the default status of notebook permission setting. As you can see in the placeholder : 
screen shot 2016-06-07 at 1 10 06 pm

**Comment by khalidhuseynov** [comment]
LGTM

**Comment by corneadoug** [comment]
No problem of rendering with the new "theme"
Merging if there is no more discussions

**Pull request closed without merging**